frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -196,7 +249,12 @@ class Sidebar extends PureComponent<SidebarProps, State> {
   // BUG(tvst): X button should have same color as hamburger.
   // BUG(tvst): X and > buttons should have same margins as hamburger.
```

Comment:
Hoping to get your feedback on these - as far as I can tell the X and > sidebar buttons have same color & margins as the hamburger menu button, can we remove these TODO items?

Reply:
Yeah, that's correct! We fix that a while back on a different papercut. Removed the comments on [773fa1b](https://github.com/streamlit/streamlit/pull/6050/commits/773fa1bcf187fc49d7310c3fd17980db2afdb448)